doc/en/how-to/usage.rst

```diff
@@ -168,15 +168,17 @@ You can invoke ``pytest`` from Python code directly:
 
 .. code-block:: python
 
-    pytest.main()
+    if __name__ == "__main__":
+        sys.exit(pytest.main())
 
 this acts as if you would call "pytest" from the command line.
 It will not raise ``SystemExit`` but return the exitcode instead.
```

Comment:
While at it lets add some useful links. 👍 

```suggestion
It will not raise :class:`SystemExit` but return the :ref:`exit code <exit-codes>` instead.
```